sdk/communication/azure-communication-networktraversal/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0-beta.3 (Unreleased)
-
-### Features Added
+## 1.1.0-beta.3 (2024-04-11)
 
 ### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Please note, this service is shutting down on 03/31/2024 and there is no replacement, please read our [retirement notice](https://azure.microsoft.com/updates/retirement-notice-azure-communication-services-network-traversal-turn-public-preview-is-retiring/) for more details.
```

Comment:
Since the release of this library is happening after the shutdown date, we should update this to say "this service was shut down on 03/31/2024..."

Reply:
updated